src/main/kotlin/com/cjbooms/fabrikt/generators/GeneratorUtils.kt

```diff
@@ -96,20 +108,28 @@ object GeneratorUtils {
     fun RequestBody.getPrimaryContentMediaType(): Map.Entry<String, MediaType>? =
         this.contentMediaTypes.entries.firstOrNull()
 
-    fun Response.getPrimaryAcceptMediaType(): Map.Entry<String, MediaType>? =
+    fun Response.getPrimaryContentMediaType(): Map.Entry<String, MediaType>? =
         this.contentMediaTypes.entries.firstOrNull()
 
-    fun Operation.getBodyResponses(): List<Response> =
-        this.responses.filter { it.key != "default" }.values.filter(Response::hasContentMediaTypes)
+    fun Response.isMultiContentMediaType() = this.contentMediaTypes.entries.size > 1
+
+    fun Operation.firstResponse() = this.getBodyResponses().firstOrNull()
 
-    fun Operation.getPrimaryAcceptMediaType(): Map.Entry<String, MediaType>? =
-        this.getBodyResponses().map { response -> response.getPrimaryAcceptMediaType() }.firstOrNull()
+    fun Operation.getPrimaryContentMediaType(): Map.Entry<String, MediaType>? =
+        this.getBodyResponses().map { response -> response.getPrimaryContentMediaType() }.firstOrNull()
+
+    fun Operation.defaultContentMediaType() = this.firstResponse()?.getPrimaryContentMediaType()?.key ?: "application/json"
+
+    fun Operation.isMultiContentMediaType() = this.firstResponse()?.isMultiContentMediaType()
```

Comment:
These two function names are a bit overloaded:
1. `defaultContentMediaType` Reads to me as the problem json type when defaulted. Maybe primary again? `getPrimaryContentMediaTypeKey`
2. `isMultiContentMediaType` Reads like we could be checking for a 207 multi response type. Maybe `hasMultipleContentMediaTypes`


Also, please add a return type to these public functions.